tests/test_fixtures.py

```diff
@@ -43,12 +43,14 @@ def test_env_old_fixture(short_env, short_test):
         assert spawn.exitstatus == 0
 
 def test_env_env_fixture(short_env, short_test):
-    env=os.environ.copy()
-    env['LG_ENV'] = short_env
+    env=os.environ
+    env['LG_ENV'] = str(short_env)
+    print(env)
```

Comment:
Leftover debug print? Please also describe what was broken and how this change fixes the problem.